cmd/slinky/main.go

```diff
@@ -289,7 +289,11 @@ func runOracle() error {
 	defer orch.Stop()
 
 	// Create the oracle and start the oracle server.
-	oracleOpts = append(oracleOpts, oracle.WithProviders(orch.GetPriceProviders()))
+	oracleOpts = append(
+		oracleOpts,
+		oracle.WithProviders(orch.GetPriceProviders()),
+		oracle.WithMarketMapGetter(orch.GetMarketMap),
```

Comment:
Why are we making this optional?

I think we should just pass a reference to the `orchestrator` into the `oracle` upon construction and use the `GetMarketMap` function directly.  There should only ever be one implementation of this `MarketMapGetter` function.

Reply:
This is kind of a broader discussion about how we construct the oracle in general.  Right now the oracle does not "own" or have a notion of the orchestrator, which seems confusing to me

Reply:
yeah, that was confusing to me as well. it seemed the oracle needs to know about things the orchestrator is doing, so i thought id find a direct ref to it but found the options instead

i can try to address that in this PR if we'd like or can open a new linear ticket?

Reply:
yea ideally we move away from the price oracle and move core logic into the orchestrator and make it the central info maintainer. Definitely not in scope for this PR, would open a separate issue.

Reply:
agreed - let's open a separate issue for refactoring the logic.


I think for the meantime, maybe we can just pass a ref to the orchestrator and always enable queries?

Reply:
ticket [here](https://linear.app/skip/issue/BLO-1329/move-oracle-service-into-orchestrator), marking this as resolved